target/linux/ath79/image/generic.mk

```diff
@@ -2539,6 +2539,19 @@ define Device/teltonika_rut955-h7v3c0
 endef
 TARGET_DEVICES += teltonika_rut955-h7v3c0
 
+define Device/trendnet_tew-673gru
+  SOC := ar7161
+  DEVICE_VENDOR := Trendnet
+  DEVICE_MODEL := TEW-673GRU
+  DEVICE_VARIANT := v1.0R
+  IMAGE_SIZE := 6208k
+  IMAGE/sysupgrade.bin := append-kernel | append-rootfs | pad-rootfs | \
+		check-size | append-metadata
+  DEVICE_PACKAGES := kmod-usb-ohci kmod-usb2 kmod-owl-loader
```

Comment:
need to add package `kmod-switch-rtl8366s` here https://github.com/openwrt/openwrt/commit/6e0f0eae5b48bddead035d9dd89b710c10c50337